CI/PHPUnit/run_tests_with_reporting.sh

```diff
@@ -54,37 +54,25 @@ if [[ -e "$PHPUNIT_RESULTS_PATH" ]]
 				FAILURE=true
 		fi
 
-		if [[ "$TRAVIS_EVENT_TYPE" != "pull_request" ]]
+		if [[ "$TRAVIS_EVENT_TYPE" == "pull_request" ]]
```

Comment:
@lauraquellmalz 
Why did you change the logic of this if statement? The results should only be committed if this is **NOT** a pull request.

Reply:
Ups, sorry. Fixed.